code/go/0chain.net/smartcontract/dbs/event/reward_test.go

```diff
@@ -56,26 +56,26 @@ func TestRewardEvents(t *testing.T) {
 		StartBlock: 0,
 		EndBlock:   900,
 	}
-	claimedReward, err := eventDb.GetRewardClaimedTotal(rewardQuery)
+	claimedReward, err := eventDb.GetRewardClaimedTotalBetweenBlocks(rewardQuery)
 	require.NoError(t, err, "Error while getting sum of rewards")
-	require.Equal(t, int64(1500), claimedReward, "Not all rewards were calculated")
+	require.Equal(t, int64(1000), claimedReward, "Not all rewards were calculated")
```

Comment:
why do these values changed?

Reply:
I think it was just wrong to begin with. This testfile was added for local testing it seems and doesn't run on CI. This line [here](https://github.com/0chain/0chain/pull/1586/files/00041a9287f476383b897cb5259560fe35682a9e#diff-ce50bde40e83670d0ecf2b21577ec4664d68a6285a613643a4c2a38b50311704L13)